src/migrations/20230208091810-assign-project-api-token-permissions_admin.js

```diff
@@ -0,0 +1,23 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            INSERT INTO role_permission (role_id, permission_id)
+            SELECT (SELECT id as role_id from roles WHERE name = 'Admin' LIMIT 1),
+                   p.id  as permission_id
+            FROM permissions p
+            WHERE p.permission IN
+                  ('READ_PROJECT_API_TOKEN',
+                   'CREATE_PROJECT_API_TOKEN',
+                   'DELETE_PROJECT_API_TOKEN');
+        `,
+        cb,
+    );
+};
```

Comment:
I don't think the admin needs to have these permissions, because the ADMIN permission will short circuit the permissions checks.